oemer/bbox.py

```diff
@@ -47,14 +52,17 @@ def merge_nearby_bbox(bboxes, distance, x_factor=1, y_factor=1):
     return new_box
 
 
-def rm_merge_overlap_bbox(bboxes, mode='remove', overlap_ratio=0.5):
+def rm_merge_overlap_bbox(
+        bboxes: List[BBox], 
+        mode: str = 'remove', 
+        overlap_ratio: float = 0.5) -> Union[List[BBox], List[BBox], List[BBox]]:
     assert mode in ['remove', 'merge'], mode
 
     pts = np.array([(box[2], box[3]) for box in bboxes])
     max_x, max_y = np.max(pts[:, 0]), np.max(pts[:, 1])
     mask = np.zeros((max_y, max_x), dtype=np.uint16)
 
-    box_info = []
+    box_info: Any = []
```

Comment:
If you already assign it as an empty list, then you should annotated it as `box_info: List[Any]`, or even better is to observe what will be in the list. In this case the `box_info` should be annotated as `box_info: List[Dict[str, Any]]`